src/js/containers/account/filters/AccountProgramActivityContainer.jsx

```diff
@@ -0,0 +1,131 @@
+/**
+ * AccountProgramActivityContainer.jsx
+ * Created by michaelbray on 4/14/17.
+ */
+
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { isCancel } from 'axios';
+
+import * as accountFilterActions from 'redux/actions/account/accountFilterActions';
+import * as AccountHelper from 'helpers/accountHelper';
+
+import ProgramActivityFilter from
+    'components/account/filters/programActivity/ProgramActivityFilter';
+
+const propTypes = {
+    setAvailableProgramActivities: React.PropTypes.func,
+    toggleProgramActivity: React.PropTypes.func,
+    account: React.PropTypes.object
+};
+
+export class AccountProgramActivityContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            noResults: false
```

Comment:
Can we also track the in flight status of the API call instead of showing old data/an empty div?